src/toolchain/toolchain.ts

```diff
@@ -575,6 +579,42 @@ export class SwiftToolchain {
         return configuration.sdk !== "" ? configuration.sdk : undefined;
     }
 
+    /**
+     * @returns path to the swiftpm-testing-helper binary, if it exists.
+     */
+    private static async getSwiftPMTestingHelperPath(): Promise<string | undefined> {
+        if (process.platform === "darwin") {
+            const developerDir = await this.getXcodeDeveloperDir(
+                configuration.swiftEnvironmentVariables
+            );
+            const swiftPMHelperPath = path.join(
```

Comment:
should this not be the selected toolchain?

Reply:
I believe you're right.

Reply:
Updated to look in the `toolchainPath` which is either the users selected toolchain or their Xcode, whichever they have configured.